docs/config/README.md

```diff
@@ -3601,6 +3602,39 @@ detect_extensions = []
 python_binary = ['./venv/bin/python', 'python', 'python3', 'python2']
 ```
 
+## Quarto
+
+The `quarto` module shows the current installed version of Quarto used in a project.
+
+By default, the module will be shown if any of the following conditions are met:
+
+- The current directory contains a `_quarto.yml` file
+- The current directory contains any `*.qmd` file
+
+### Options
+
+| Option              | Default                              | Description                                                               |
+| ------------------- | ------------------------------------ | ------------------------------------------------------------------------- |
+| `format`            | `'via [$symbol($version )]($style)'` | The format for the module.                                                |
+| `version_format`    | `'v${raw}'`                          | The version format. Available vars are `raw`, `major`, `minor`, & `patch` |
+| `symbol`            | `'⨁ '`                               | A format string representing the symbol of Quarto                         |
+| `style`             | `'bold #0093A7'`                     | The style for the module.                                                 |
+| `detect_extensions` | `['.qmd']`                           | Which extensions should trigger this module.                              |
+| `detect_files`      | `['_quarto.yml']`                    | Which filenames should trigger this module.                               |
+| `detect_folders`    | `[]`                                 | Which folders should trigger this module.                                 |
+| `disabled`          | `false`                              | Disables the `quarto` module.                                             |
+
+### Variables
+
+| Variable      | Example   | Description                                       |
+| ------------- | --------- | ------------------------------------------------- |
+| version       | `1.4.549` | The version of `quarto`, alias for quarto_version |
+| typst_version | `default` | The current Typst version                         |
```

Comment:
I don'T see anything parsing/filling that variable. Is this needed/wanted?

In any case: given that quarto bundles typst, what's the value in this, given that in most (or even all cases: can you overwrite the typst version by installing a newer version), the quarto version will determine the typst version?

Reply:
Indeed, that's a copy/paste fail. I removed the variable.